sqlalchemy_history/utils.py

```diff
@@ -390,3 +390,11 @@ def replace(self, col):
 
 def adapt_columns(expr):
     return VersioningClauseAdapter().traverse(expr)
+
+def get_association_proxy_mapping(klass):
```

Comment:
Rename to `get_association_proxies`
Use `for key, prop` in the for loop below - key is normally a string.